Mockoon.Dockerfile

```
@@ -0,0 +1,27 @@
+
+##compilation
+RUN tsc mockoon/data.ts --outDir out --rootDir ./ \
+  --module NodeNext --moduleResolution NodeNext \
+  --target ESNext --strict --resolveJsonModule \
+  --experimentalDecorators --emitDecoratorMetadata \
+  --esModuleInterop --allowSyntheticDefaultImports \
+  --skipLibCheck --noImplicitAny --noEmitOnError \
+  --lib es2018,dom
+
+##create data.json
+RUN node ./out/mockoon/data.js
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)